tests/test_lab.py

```diff
@@ -92,12 +93,12 @@ def has_debug_params(self) -> bool:
     Command(("model", "test")),
     Command(("model", "train")),
     Command(("model", "list")),
-    Command(("data",), needs_config=False),
+    Command(("data",), needs_config=False, should_fail=False),
```

Comment:
is this change needed in this PR if we never test the `should_fail=True` use case in this PR?

Reply:
seems to me like `should_fail` would default to false even if not specified so all the cases are probably defaulting to that right now so im having trouble understanding what this change is adding

Reply:
should_fail is default True above.

Reply:
oh sorry, missed that 
though i feel like thats an odd default tbh

Reply:
i think it might be a bit more intuitive to inverse it personally
not a blocker though